### PR TITLE
S3 + S4

### PR DESCRIPTION
**Pull request by MikeLippincott**
This PR updates 2 figures

### CODE REVIEW

File: figures/S3.supplemental_figure3/figures/final_figure_pt2.png
Comment (axiomcura) [inline figures/S3.supplemental_figure3/figures/final_figure_pt2.png:1]: Really like the changes! Much clearer to read. Really nice to see how the Pyroptosis clusters very well !

**Review by axiomcura — APPROVED** [review]
LGTM! Nice to see some beautiful plots! Also, I left a small comment, make sure to take a peek when you have the chance!

File: 1.Exploratory_Data_Analysis/notebooks/9.subset_umap.ipynb [hunk 1]
```diff
@@ -527,7 +527,7 @@
                     "name": "stderr",
```
Comment (axiomcura) [inline 1.Exploratory_Data_Analysis/notebooks/9.subset_umap.ipynb:527]: <p>The variable names are getting a bit too hard to follow. What does a_h, a_p, and h_p mean? Consider updating the varaible names to improve understanding what you are doing. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/118/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (axiomcura) [inline 1.Exploratory_Data_Analysis/notebooks/9.subset_umap.ipynb:527]: <p>same comment above</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/118/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

File: figures/S3.supplemental_figure3/figures/final_figure_pt1.png
[the remark this answers is not in the document]
Comment (MikeLippincott) [reply figures/S3.supplemental_figure3/figures/final_figure_pt1.png:1]: I think it is and I have to save a figure as png to import it as a patchwork object - ... best fix I could find for this as complexheatmap objects are not compatible with ggplot objects
